project/project.go

```diff
@@ -226,6 +226,35 @@ func (p *Project) Pull(services ...string) error {
 	}), nil)
 }
 
+// ListStoppedContainers lists the stopped containers for the specified services.
+func (p *Project) ListStoppedContainers(services ...string) ([]string, error) {
+	stoppedContainers := []string{}
+	err := p.forEach(services, wrapperAction(func(wrapper *serviceWrapper, wrappers map[string]*serviceWrapper) {
+		wrapper.Do(nil, EventServiceDeleteStart, EventServiceDelete, func(service Service) error {
+			containers, innerErr := service.Containers()
+			if innerErr != nil {
+				return innerErr
+			}
+
+			for _, container := range containers {
+				running, innerErr := container.IsRunning()
+				if innerErr != nil {
+					log.Error(innerErr)
+				}
+				if !running {
+					stoppedContainers = append(stoppedContainers, container.Name())
```

Comment:
@dnephin it could be the `ID` but both names and ids are unique so I feels it would do the same (but yeah using `ID` is ok, I'll update).